package.json

```diff
@@ -30,6 +30,7 @@
         "new-version": "lerna version --conventional-commits --no-push"
     },
     "devDependencies": {
-        "lerna": "^3.22.1"
+        "lerna": "^3.22.1",
+        "svelte-check": "^2.2.6"
```

Comment:
Why do we using the "next" version of this plugin, not the last one? Are there any features we use that are missing in this one?